apps/frontend/components/til/til-calendar/til-calendar.tsx

```
@@ -0,0 +1,126 @@
+        } else if (tilEntry.status === 'new') {
+          cellClass += ' bg-blue-100 rounded-full';
+          statusElement = (
+            <div className="absolute bottom-1 left-1/2 transform -translate-x-1/2 text-xs bg-red-500 text-white rounded-full w-4 h-4 flex items-center justify-center">
+              N
+            </div>
+          );
+        }
+      }
+
+      days.push(
+        <div key={day} className={cellClass} onClick={() => onDateClick && onDateClick(date)}>
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)